blackjax/util.py

```diff
@@ -136,3 +137,47 @@ def index_pytree(input_pytree: ArrayLikeTree) -> ArrayTree:
     (dim_input,) = flat_input.shape
     array = jnp.arange(dim_input, dtype=flat_input.dtype)
     return unravel_fn(array)
+
+
+def run_inference_algorithm(
+    rng_key: PRNGKey,
+    initial_state_or_position: ArrayLikeTree,
+    inference_algorithm: Union[SamplingAlgorithm, VIAlgorithm],
+    num_steps: int,
+) -> Tuple[State, State, Info]:
+    """Wrapper to run an inference algorithm.
+
+    Parameters
+    ----------
+    rng_key : PRNGKey
```

Comment:
```suggestion
    rng_key:
```

Since we already type the input, no need to repeat the type here. 
Same for the rest of the args.

Reply:
@junpenglao any thoughts on my above comment about the "bug" in the tests due to using a dummy `logdensity_fn`?